transactions/README.md

```diff
@@ -17,7 +17,39 @@ This allows us to implement **Write-Ahead Logging** (WAL):
 ### Concurrency Control
 **Schedule** for transactions is an ordering of the operations performed by those transaction. We want to find two equivalent schedules, i.e. they order all **conflicting** operations in the same way.
 
+**Serial Schedule**: a serial execution of two (or more) transactions.  
+![serial schedule image](/transactions/serial_schedule.png)
+
+**Conflict Serializable**: when a schedule is equivalent to **some** serial schedule
+
+But how do we test for serializability? We create a **precedence graph**, where each node **t** represents a transaction **t**, and there is an edge from **s** to **t** if some action **s** precedes and conflicts with some action of **t**. Then, a schedule is **conflict-serializable** if and only if its precedence graph is **acyclic.** 
```

Comment:
if you're refering to a code entity (e.g. s or t), can you use back ticks `s`, `t`? only vocab should be bold.